config/initializers/sentry.rb

```diff
@@ -1,5 +1,11 @@
 # Configure sentry
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = ENV['SENTRY_DSN']
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+  config.send_default_pii = true
+
+  # Roughly reimplement sanitize_fields; see https://github.com/getsentry/sentry-ruby/issues/1140#issuecomment-788089679
```

Comment:
big thanks to the sentry staffer in https://github.com/getsentry/sentry-ruby/issues/1140 for this snippet.